package.json

```diff
@@ -31,64 +31,68 @@
   },
   "dependencies": {
     "@expo-google-fonts/source-sans-pro": "0.2.2",
-    "@expo/cli": "^0.4.11",
-    "@react-native-async-storage/async-storage": "1.17.7",
+    "@expo/cli": "0.4.11",
+    "@react-native-async-storage/async-storage": "1.17.11",
     "@react-native-community/masked-view": "0.1.11",
     "@react-navigation/native": "6.0.11",
     "@react-navigation/stack": "6.2.2",
     "@rneui/base": "4.0.0-rc.5",
     "@rneui/themed": "4.0.0-rc.5",
-    "expo-application": "4.1.0",
-    "expo-constants": "13.1.1",
-    "expo-device": "4.2.0",
-    "expo-font": "10.1.0",
-    "expo-image-picker": "13.1.1",
-    "expo-linear-gradient": "11.3.0",
-    "expo-linking": "^3.1.0",
-    "expo-permissions": "13.2.0",
-    "expo-splash-screen": "0.15.1",
-    "expo-status-bar": "1.3.0",
-    "expo-updates": "0.13.3",
+    "expo": "47.0.13",
+    "expo-application": "5.0.1",
+    "expo-constants": "14.0.2",
+    "expo-device": "5.0.0",
+    "expo-font": "11.0.1",
+    "expo-image-picker": "14.0.2",
+    "expo-linear-gradient": "12.0.1",
+    "expo-linking": "3.1.0",
+    "expo-permissions": "14.0.0",
+    "expo-splash-screen": "0.17.5",
+    "expo-status-bar": "1.4.2",
+    "expo-updates": "0.15.6",
     "formik": "2.2.9",
     "mime": "3.0.0",
-    "react": "17.0.2",
-    "react-dom": "17.0.2",
-    "react-native": "0.68.2",
+    "react": "18.1.0",
+    "react-dom": "18.1.0",
+    "react-native": "0.70.5",
     "react-native-dropdown-picker": "5.4.2",
-    "react-native-gesture-handler": "2.2.1",
-    "react-native-reanimated": "2.8.0",
-    "react-native-render-html": "^6.3.4",
-    "react-native-safe-area-context": "4.2.4",
-    "react-native-screens": "3.11.1",
-    "react-native-web": "0.17.7",
-    "sentry-expo": "4.2.0",
+    "react-native-gesture-handler": "2.8.0",
+    "react-native-reanimated": "2.12.0",
+    "react-native-render-html": "6.3.4",
+    "react-native-safe-area-context": "4.4.1",
+    "react-native-screens": "3.18.0",
+    "react-native-web": "0.18.9",
+    "sentry-expo": "5.0.3",
     "yup": "0.32.11"
   },
   "devDependencies": {
     "@babel/core": "7.18.6",
     "@babel/plugin-proposal-export-namespace-from": "7.18.6",
     "@babel/preset-env": "7.18.6",
     "@babel/preset-react": "7.18.6",
-    "@expo/webpack-config": "0.16.25",
+    "@expo/webpack-config": "18.0.1",
     "@testing-library/jest-native": "4.0.5",
     "@testing-library/react-native": "10.1.1",
     "eslint": "8.19.0",
     "eslint-plugin-jest": "26.5.3",
     "eslint-plugin-react": "7.30.1",
     "eslint-plugin-react-native": "4.0.0",
     "eslint-plugin-react-native-a11y": "3.2.1",
-    "eslint-plugin-simple-import-sort": "^7.0.0",
+    "eslint-plugin-simple-import-sort": "7.0.0",
     "husky": "8.0.1",
     "jest": "28.1.2",
     "jest-expo": "45.0.1",
     "lint-staged": "13.0.3",
     "patch-package": "6.4.7",
```

Comment:
maybe we can remove this package and the file in `patches/...` ?

Reply:
apparently yes :)

Reply:
@phillimorland let's remove it then?

Reply:
sorry, didn't push it

Reply:
@khamui there now

Reply:
issue: https://github.com/liqd/zt-app/issues/411